include/nbl/ext/RadixSort/RadixSort.h

```diff
@@ -0,0 +1,130 @@
+#ifndef _NBL_EXT_RADIXSORT_INCLUDED_
+
+#include "nabla.h"
+#include "nbl/ext/Scan/Scan.h"
+
+namespace nbl
+{
+namespace ext
+{
+namespace RadixSort
+{
+
+using ScanClass = ext::Scan::Scan<uint32_t>;
+
+typedef uint32_t uint;
+#include "nbl/builtin/glsl/ext/RadixSort/parameters_struct.glsl"
+
+class RadixSort final : public core::IReferenceCounted
+{
+public:
+	_NBL_STATIC_INLINE_CONSTEXPR uint32_t DEFAULT_WORKGROUP_SIZE = 256u;
+	_NBL_STATIC_INLINE_CONSTEXPR uint32_t BITS_PER_PASS = 4u;
```

Comment:
these should be tweakable at runtime in the future I guess, especially if you want to investigate how much the perf improves

`const` but not `constexpr`